langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicStreamingChatModel.java

```diff
@@ -30,4 +37,10 @@ public enum Types {
             this.value = modelID;
         }
     }
+
+    @Override
+    public void generate(List<ChatMessage> messages, StreamingResponseHandler<AiMessage> handler) {
```

Comment:
same here

Reply:
done for all generate methods

<img width="756" alt="image" src="https://github.com/user-attachments/assets/85a58ff6-9abe-4fe8-a7e9-a44f30063fd0">